cmd/podman/common.go

```diff
@@ -401,6 +401,14 @@ func getCreateFlags(c *cliconfig.PodmanCommand) {
 		"publish-all", "P", false,
 		"Publish all exposed ports to random ports on the host interface",
 	)
+	createFlags.Bool(
+		"pull-always", false,
+		"Always pull the image from the source repository",
+	)
+	createFlags.Bool(
+		"pull", false,
+		"Update the image if it exists in storage but is older than its source (default false)",
+	)
```

Comment:
Sorry if a dup comment, thought I'd left one here.  In Buildah we use:
```
fs.BoolVar(&flags.Pull, "pull", true, "pull the image if not present")
fs.BoolVar(&flags.PullAlways, "pull-always", false, "pull the image, even if a version is present")
```
I'd suggest similar verbiage and flipping pull and pull-always order in the code.

Reply:
I'll copy the --pull-always, but --pull updates the image if it is out of date, and fails if its not present.

Reply:
On Buildah --pull works like I described.  Pulls if it is not there, and checks for newer if it is there.
--pull=false means fail if it is not local.

Reply:
changed to update :)